components/ParsePOSPayment/Receive-Invoice.tsx

```diff
@@ -248,6 +298,59 @@ function ReceiveInvoice({ recipientWalletCurrency, walletId, state, dispatch }:
         paymentAmount={paymentAmount}
         dispatch={dispatch}
       />
+      {openModal && (
+        <Modal
+          show={openModal}
+          onHide={() => setOpenModal(false)}
+          aria-labelledby="contained-modal-title-vcenter"
+          centered
+        >
+          <Modal.Header className={styles.modal_header} closeButton>
+            You can payment with
```

Comment:
This should be
```suggestion
            You can make payment with
```
or 
```
You can pay with
```